src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -304,7 +304,13 @@ IECoreGL::ConstRenderablePtr StandardLightVisualiser::visualise( const IECore::I
 	else
 	{
 		// Treat everything else as a point light.
-		ornaments->addChild( const_pointer_cast<IECoreGL::Renderable>( pointRays() ) );
+		const float radius = parameter<float>( metadataTarget, shaderParameters, "radiusParameter", -1 );
```

Comment:
The visualisation is broken for all point lights except the Arnold one. I suspect this is due to the default of `-1` - can you verify, and use a default of 0 to fix it?

Reply:
8d49e15